frontend/javascripts/oxalis/model/accessors/dataset_layer_transformation_accessor.ts

```diff
@@ -285,7 +284,7 @@ function _getTransformsForLayerThatDoesNotSupportTransformationConfigOrNull(
     return null;
   }
 
-  return toIdentityTransformMaybe(invertTransform(transformsOfNativeLayer));
```

Comment:
are you sure that `toIdentityTransformMaybe` does not need to be called? I'm wondering because `invertTransform` always returns a new object, but maybe `toIdentityTransformMaybe` happens on the call site? to be honest, it is a bit hard for me to follow where exactly the identity-stuff happens. however, I don't have a great idea for how to improve this.

Reply:
Oh you are totally correct. `toIdentityTransformMaybe` should be kept here :+1: